src/Console/WorkCommand.php

```diff
@@ -8,6 +8,8 @@
 
 class WorkCommand extends IlluminateWorkCommand
 {
+    const SIGNATURE_REGEX_PATTERN = '/([\w\:]+)(?=\s|\{)/i';
```

Comment:
Oh actually. Yea. When used in the tests it captures the custom name too. So Ill add the hypen in too. What about numbers?